docs/flow/modin/core/storage_formats/pandas/parsers.rst

```diff
@@ -16,3 +16,8 @@ be returned.
 .. note:: 
     If you are interested in the data parsing mechanism implementation details, please refer
     to the source code documentation.
+
+Examples
+--------
+- .. autoclass:: modin.core.storage_formats.pandas.parsers.PandasCSVParser
+- .. autoclass:: modin.core.storage_formats.pandas.parsers.PandasCSVGlobParser
```

Comment:
The same as described in https://github.com/modin-project/modin/pull/3819#discussion_r767549234